tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -96,7 +96,7 @@ suite('Language server validation', async () => {
             await ide.waitStatusBarContains('Activating Language Support for Java');
         }
 
-        await ide.waitStatusBarTextAbsence('Activating Language Support for Java', 1800000);
+        await ide.waitStatusBarTextAbsence('Activating Language Support for Java', 1_800_000); // todo: are you sure you want to wait half an hour for the LS to initialize?
```

Comment:
The happy path test language server was given 30 minutes to start (1 800 000 ms). I believe this was a typo instead of the intended 3 minutes (180 000 ms).

Reply:
I agree that 30 minutes is too much but 3 minutes is too small, note that Che CI instance may has a low performance, so I suggest putting 15 minutes here.

Reply:
thank you for the input

Reply:
Fixed as per discussion.